lib/bibliothecary/parsers/maven.rb

```diff
@@ -24,6 +30,10 @@ def self.mapping
             kind: 'lockfile',
             parser: :parse_ivy_report
           },
+          /^tidelift-gradle-resolved\.txt|.*\/tidelift-gradle-resolved\.txt$/i => {
```

Comment:
let’s not put tidelift in the name here - maybe “gradle-dependencies-q.txt” is exactly what it is?